Attributed/UIFont.swift

```diff
@@ -32,33 +32,48 @@ extension UIFont {
     }
     
     func fontWithMonospacedNumbers() -> UIFont {
-        return UIFont(descriptor: fontDescriptor.addingAttributes([
-            UIFontDescriptorFeatureSettingsAttribute: [
-                [
-                    UIFontFeatureTypeIdentifierKey: kNumberSpacingType,
-                    UIFontFeatureSelectorIdentifierKey: kMonospacedNumbersSelector,
-                ],
-            ],
-            ]), size: pointSize)
+        #if os(watchOS)
```

Comment:
Cool, I didn't know about this method!

Should be able to merge this check and the iOS availability check. Something like:

    if #available(iOS 9.0, watchOS 2.0, *) {